x/ibc/04-channel/client/cli/query.go

```diff
@@ -0,0 +1,75 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	cli "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
+)
+
+// TODO: get proofs
+// const (
+// 	FlagProve = "prove"
+// )
+
+// GetQueryCmd returns the query commands for IBC channels
+func GetQueryCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
+	ics04ChannelQueryCmd := &cobra.Command{
+		Use:                "channel",
+		Short:              "IBC channel query subcommands",
+		DisableFlagParsing: true,
+	}
+
+	ics04ChannelQueryCmd.AddCommand(cli.GetCommands(
+		GetCmdQueryChannel(storeKey, cdc),
+	)...)
+
+	return ics04ChannelQueryCmd
+}
+
+// GetCmdQueryChannel defines the command to query a channel end
+func GetCmdQueryChannel(storeKey string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "end [port-id] [channel-id]",
```

Comment:
Need more explanatory text here. Make it explicit in terms of dest chain and src chain.